Tests/Admin/BaseBlockAdminTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\PageBundle\Tests\Admin;
+
+class BaseBlockAdminTest extends \PHPUnit_Framework_TestCase
```

Comment:
please extends from this class:
https://github.com/sonata-project/SonataPageBundle/blob/3.x/Tests/Helpers/PHPUnit_Framework_TestCase.php

Reply:
@OskarStark got it, done

Reply:
Thank you!